loopy/codegen/control.py

```diff
@@ -142,13 +144,19 @@ def generate_code_for_sched_index(codegen_state, sched_index):
 
         from loopy.kernel.data import (UnrolledIlpTag, UnrollTag,
                 ForceSequentialTag, LoopedIlpTag, VectorizeTag,
-                InOrderSequentialSequentialTag, filter_iname_tags_by_type)
+                InOrderSequentialSequentialTag, OpenMPSIMDTag,
+                filter_iname_tags_by_type)
         if filter_iname_tags_by_type(tags, (UnrollTag, UnrolledIlpTag)):
             func = generate_unroll_loop
         elif filter_iname_tags_by_type(tags, VectorizeTag):
-            func = generate_vectorize_loop
+            from loopy.target.c import CVecTarget
```

Comment:
So there a lot of files with little amount of changes. All of them are in the form of 
`(kernel.iname_tags_of_type(iname, VectorizeTag) and isinstance(kernel.target, CVecTarget))`.
This is code that I added when I tried to go through the `vec` tag and I ran into issues

so that happens in this file which is 
`loopy/codegen/control.py`
and also in
```
loopy/codegen/bounds.py
loopy/codegen/loop.py
loopy/kernel/__init__.py
loopy/kernel/array.py
loopy/schedule/__init__.py
```

I think the issues were caused by the fact that I made `VectorizeTag` inherit from `HardwareConcurrentTag` here
`loopy/kernel/data.py`